### PR TITLE
Initial work on sys_usbd syscalls

### PR DESCRIPTION
**Pull request by hcorion**
Whew, this took way to long.

Do note, that you won't be able to plug in usb devices and have RPCS3 use them yet, if you're gonna test this, then just test for regressions.

I'll do a more in-depth description in the monthly progress report.

### CODE REVIEW

File: rpcs3/Emu/Cell/lv2/sys_usbd.h [hunk 1]
```diff
@@ -1,18 +1,54 @@
...
 // SysCalls
 
-s32 sys_usbd_initialize();
+struct DeviceListUnknownDataType
+{
+	u8 unk1;
+	u8 deviceID;
+	u8 unk3;
+	u8 unk4;
+} ;
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/lv2/sys_usbd.h]: weird space.

and the tab-spacings in the next two structs seem off

xD

**Comment by Marthgun** [comment]
Should some of these lines be trimmed? in sys_prx.cpp, i.e. something like this,
```
unsigned char bytes[] = {0x12, 0x1, 0x0, 0x2, 0x0, 0x0, 0x0, 0x40, 
                         0x4c, 0x5, 0x50, 0x2, 0x9, 0x0, 0x3, 
                         0x4, 0x5, 0x1, 0x9, 0x2, 0x20, 0x0,...........
```  

i'm just asking,  which is why no review comment

**Comment by hcorion** [comment]
Yes, I will @Marthgun , my editor does it automatically, so it didn't bother me.

File: rpcs3/Emu/Cell/lv2/sys_usbd.h (hunk 1 above)
Comment (hcorion) [reply rpcs3/Emu/Cell/lv2/sys_usbd.h]: ah, good ol' copy-pasta.

Addressed.

**Pull request merged**